keyboards/at_at/660m/readme.md

```diff
@@ -0,0 +1,13 @@
+# AT-AT 660M
+
+660M keyboard
+
+Keyboard Maintainer: [Adrien Tétar](https://github.com/adrientetar)  
+Hardware Supported: AT-AT 660M
+
```

Comment:
The Hardware Availability information (links to vendors or a repository if you're open-sourcing the hardware) should go here on Line 7 when you're ready.

Everything else looks good to me.